src/main/webapp/util/Storage.js

```diff
@@ -0,0 +1,38 @@
+const CurrentUser = "CurrentUser";
+const UserIsLogged = "isLogged";
+const URLFrom = "URLFrom";
+
+export const Storage = {
+
+  clearStorage: () => {
+    localStorage.clear();
+  },
+
+  setCurrentUser: data => {
+    localStorage.setItem(CurrentUser, JSON.stringify(data));
+  },
+
+  getCurrentUser: () => {
+    return localStorage.getItem(CurrentUser) ? JSON.parse(localStorage.getItem(CurrentUser)) : null;
+  },
+
+  userIsLogged: () => {
+    return localStorage.getItem(UserIsLogged) === 'true';
+  },
+
+  setUserIsLogged: value => {
+    localStorage.setItem(UserIsLogged, value);
+  },
+
+  setLocationFrom: urlFrom => {
+    localStorage.setItem(URLFrom, JSON.stringify(urlFrom));
+  },
+
+  getLocationFrom: () => {
+    return localStorage.getItem(URLFrom) ? JSON.parse(localStorage.getItem(URLFrom)) : null;
+  },
+
+  removeLocationFrom: () => {
+    return localStorage.removeItem(URLFrom);
+  }
+};
```

Comment:
Perhaps not for this PR, but I have some reservations about using local storage like this. I've found it to be problematic in duos. [Here's how they are doing this in Terra](https://github.com/DataBiosphere/terra-ui/blob/58d645ae53b08c7be255f199d09394f5d152a472/src/libs/browser-storage.js). I will do some research on alternatives, but for now, I'm not requesting any changes here.